### ISSUE TITLE
Documentation of the /.is choice key

### ISSUE THREAD

**Issue by ghost** [pgf-tikz/pgf#808]
Version: 3.1.4

I think that the documentation of the `/.is choice` key is unfortunate. I think the first three `/.style` keys need to go to `/.code`. That is, on p. 974 the code

```latex
\pgfkeys{/line cap/.is choice}
\pgfkeys{/line cap/round/.style={\pgfsetbuttcap}}
\pgfkeys{/line cap/butt/.style={\pgfsetroundcap}}
\pgfkeys{/line cap/rect/.style={\pgfsetrectcap}}
\pgfkeys{/line cap/rectangle/.style={/line cap=rect}}
```

should go to

```latex
\pgfkeys{/line cap/.is choice} 
\pgfkeys{/line cap/round/.code={\pgfsetroundcap}} 
\pgfkeys{/line cap/butt/.code={\pgfsetbuttcap}} 
\pgfkeys{/line cap/rect/.code={\pgfsetrectcap}} 
\pgfkeys{/line cap/rectangle/.style={/line cap=rect}}
```

**Comment by fmitha** [comment]
The "fixed" code that the bug reporter posted had two changes relative to the code that existed at that time.

The first was the change from `.style` to `.code`. The second was fixing the first two lines, which said, and still say

    \pgfkeys{/line cap/round/.style={\pgfsetbuttcap}}
    \pgfkeys{/line cap/butt/.style={\pgfsetroundcap}}

Here round should go to round, and butt to butt. But instead currently, round goes to butt and butt to round. I just checked the version of the manual on my system, and it is still as it was. I'm not sure how to handle this? Open a new issue? Reopen this one? I don't see an option to reopen.

**Comment by muzimuzhi** [comment]
@fmitha Good catch! Fixed in 425b03d.
